src/webots/nodes/utils/WbNodeOperations.cpp

```diff
@@ -310,14 +320,16 @@ static void addModelNode(QString &stream, const aiNode *node, const aiScene *sce
   const WbRotation webotsRotation(quaternion);
 
   // export the node
-  if (importAsSolid)
-    stream += " Solid { ";
-  else
-    stream += " Transform { ";
-  stream += QString(" translation %1 %2 %3 ").arg(position[0]).arg(position[1]).arg(position[2]);
-  stream += " rotation " + webotsRotation.toString(WbPrecision::FLOAT_MAX);
-  stream += QString(" scale %1 %2 %3 ").arg(scaling[0]).arg(scaling[1]).arg(scaling[2]);
-  stream += " children [";
+  if (!referenceMeshes) {
+    if (importAsSolid)
+      stream += " Solid { ";
+    else
+      stream += " Transform { ";
```

Comment:
I believe we should avoid useless spaces here:
```suggestion
      stream += " Solid {";
    else
      stream += " Transform {";
```